Libs/MRML/DisplayableManager/vtkMRMLSliceIntersectionWidget.cxx

```diff
@@ -1392,6 +1411,28 @@ bool vtkMRMLSliceIntersectionWidget::ProcessZoomSlice(vtkMRMLInteractionEventDat
   return true;
 }
 
+
+// TODO: by simzhangbest mail:dg20330034@smail.nju.edu.cn 
+// mouse left-click-drag shit between pics not smooth enough, may someone modify this
+bool vtkMRMLSliceIntersectionWidget::ProcessIncDecSlice(vtkMRMLInteractionEventData* eventData)
+{
+    // Please adjust this parameter to tune the Inc&Dec speed.
+    const double CONF = 0.01;
```

Comment:
Rather than hard-code this, it would be better to make it a function of the window size using the same code as `ProcessZoomSlice`.